pyanaconda/payload/__init__.py

```diff
@@ -32,9 +32,8 @@
 
```

Comment:
Typo in commit message 13d46bd8c0347fb7ee919ed2007051d34aa42422:
``Move the support methods for CD/DVD and ISO image installations to the the subpackage for payload``

Double ``the the``.

Reply:
Fixed.